cluster-autoscaler/integration/framework.go

```diff
@@ -245,38 +249,29 @@ func (driver *Driver) scaleWorkload(workloadName string, replicas int32) error {
 	return nil
 }
 
-func (driver *Driver) getFirstNode() (*v1.Node, error) {
+func (driver *Driver) getFirstAndLatestNode() (*v1.Node, *v1.Node, error) {
```

Comment:
better name would be `getOldestAndLatestNode`

Reply:
done